ci-operator/config/openshift/gcp-filestore-csi-driver-operator/openshift-gcp-filestore-csi-driver-operator-main.yaml

```diff
@@ -130,6 +130,14 @@ tests:
       TEST_SUITE: openshift/csi
       TRUECONDITIONS: GCPFilestoreDriverControllerServiceControllerAvailable GCPFilestoreDriverNodeServiceControllerAvailable
     workflow: openshift-e2e-gcp-csi-filestore
+- as: e2e-gcp-ovn-upgrade
+  steps:
+    cluster_profile: gcp
+    workflow: openshift-upgrade-gcp-ovn
+- as: e2e-gcp-ovn-rt-upgrade
+  steps:
+    cluster_profile: gcp-openshift-gce-devel-ci-2
+    workflow: openshift-upgrade-gcp-ovn-rt
```

Comment:
GCP filestore CSI driver is installed via OLM, i.e. it's not used anywhere in this job. I think we don't have any workflow that install the driver, upgrades the cluster and then runs some tests with the driver. Please remove the jobs, also from openshift-gcp-filestore-csi-driver repo